### PR TITLE
Manifest.json robots.txt

### PR DESCRIPTION
**Pull request by freelancing-solutions**
- Added og:properties & twitter:cards
- Added manifest.json + icons
- Added service worker 
-  Added sitemap + robots.txt

### CODE REVIEW

File: getting-started/index.html [hunk 1]
```diff
@@ -18,9 +18,10 @@
 	<!-- Meta -->
 	<title>Dogecoin - Getting Started</title>
 	<meta charset="utf-8">
-	<meta content="Getting Started with Dogecoin is as easy as 1, 2, 3! It's simple, easy to set up, and free." name="description">
 	<meta content="width=device-width, initial-scale=1.0" name="viewport">
-	
+
+	<meta content="Getting Started with Dogecoin is as easy as 1, 2, 3! It's simple,  easy to set up, and free." name="description">
```
Comment (langerhans) [inline getting-started/index.html:23]: That change is not needed.

File: index.html [hunk 2]
```diff
@@ -13,11 +13,36 @@
     -->
 
 	<!-- Meta -->
-	<title>Dogecoin</title>
+	<title>Dogecoin - #dogecoin - #doge</title>
```
Comment (langerhans) [inline index.html]: I think we can keep the webiste universal and don't need to add hashtags here.

File: index.html [hunk 3]
```diff
@@ -13,11 +13,36 @@
...
 	<meta charset="utf-8">
-	<meta content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." name="description">
 	<meta content="width=device-width, initial-scale=1.0" name="viewport">
 
+	<!-- og:properties -->
+	<meta property="og:site_name" name="dogecoin.com" />
+	<meta property="og:type" content="website" />
+	<meta property="og:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide.">
+	<meta property="og:title" content="dogecoin built on the principles of sound money - a transactional currency made for the internet" >
```
Comment (langerhans) [inline index.html]: Missing a word here

File: index.html [hunk 4]
```diff
@@ -13,11 +13,36 @@
...
+
+	<!-- Twitter Card -->
+	<meta name="twitter:card" content='summary_large_image' />
+	<meta name="twitter:site" content="@dogecoin" />
+	<meta name="twitter:creator" content="@dogecoin_dev" />
+	<meta name="twitter:title" content="#dogecoin a transactional currency built for the internet" />
+	<meta name="twitter:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+	<meta name="twitter:domain" content='https://dogecoin.com' />
+	<meta name="twitter:image" content="/imgs/doge-ogimage.png" />
+	<meta name="twitter:image:alt" contect="dogecoin.com Shiba Inu" />
```
Comment (langerhans) [inline index.html:46]: A lot of these twitter attributes are duplicated and as far as I'm aware Twitter will work fine with most of them in the og namespace. So these should be reduced to only the missing ones.

File: index.html [hunk 5]
```diff
@@ -13,11 +13,36 @@
...
+	<!--- favicon -->
+	<link rel="shortcut icon" href="../imgs/avatar.png" />
```
Comment (langerhans) [inline index.html]: Any reason for this using a relative path contrary to all the others?

File: index.html [hunk 6]
```diff
@@ -551,6 +578,13 @@ <h4 class="mobile-learn-padding" data-i18n="dgc-windows-wallet-modal-getting-sta
                   );
 </script>
 <script src="/js/dogecoin.min.js"></script>
+<script>
+	if('serviceWorker' in navigator) {
```
Comment (langerhans) [inline index.html:587]: What is the reasoning behind a service worker? I'm not a webdev but for a simple static page it seems overkill?

File: js/dogecoin-i18n.js [hunk 7]
```diff
@@ -18,10 +18,10 @@ function getUrlParameter(sParam)
 {
     var sPageURL = window.location.search.substring(1);
     var sURLVariables = sPageURL.split('&');
-    for (var i = 0; i < sURLVariables.length; i++) 
+    for (var i = 0; i < sURLVariables.length; i++)
     {
         var sParameterName = sURLVariables[i].split('=');
-        if (sParameterName[0] == sParam) 
+        if (sParameterName[0] == sParam)
```
Comment (langerhans) [inline js/dogecoin-i18n.js:24]: Changes in this file are note necessary.

**Review by langerhans — CHANGES_REQUESTED** [review]
There is also a zip file in the commit that seems to be a leftover. Please keep the changes focused on your actual goal :)

File: index.html (hunk 2 above)
Comment (freelancing-solutions) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (freelancing-solutions) [reply index.html:46]: it is not apparent to me and i am not sure of the route twitter will take however i find that twitter 
does indeed performs better with its own tags...the og: properties works best with Facebook and other platforms
the way things changes this days i would advice to keep both

File: index.html (hunk 5 above)
Comment (freelancing-solutions) [reply index.html]: its wrong my mistake..
fixed relative path

File: index.html (hunk 6 above)
Comment (freelancing-solutions) [reply index.html:587]: i am with the thought pattern that i can and will support adding more futures on dogecoin , such as forums and a  blog 
i already have code from other projects in python but i can not commit all the changes at the same time
however for now it seems you are right let me remove or comment out the service worker.

Commented out until needed, 

This was my rationale 
I have created a proposal which if successfull would mean we can upgrade dogecoin.com to using modern frameworks example react.js 
see here -> https://github.com/dogecoin/dogecoin.com/issues/126
if this proposal ends up with support from others , we would eventually need a service worker, in order to support the added functionality

File: js/dogecoin-i18n.js (hunk 7 above)
Comment (freelancing-solutions) [reply js/dogecoin-i18n.js:24]: there was no changes , could have been an auto formated that line..
however its ok.

**Comment by freelancing-solutions** [comment]
already implemented except there is a dire need for robots.txt as this site wont appear 
as needed in search engines, as without it the search engines simple skips indexing the site as they are not 
directed and or given the right to

**Pull request closed without merging**